### PR TITLE
Fixed contribution.md and pull request templating

### PR DESCRIPTION
**Pull request by Randynamic**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# See: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+./common @alber70g
+./*.*      @alber70g
+
+./common @randynamic
+./*.*      @randynamic
+
+./common @ggobugi27
+./*.*      @ggobugi27
```
Comment (alber70g) [inline .github/CODEOWNERS:11]: These can be combined to one line

```suggestion
./common @alber70g @randynamic @ggobugi27
./*.*      @alber70g @randynamic @ggobugi27 

```

And it's probably better to have a team name there. The owner of the repo can setup teams inside GitHub. Then you'd be able to say `./common @kadena-community/team` I think

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+* @kadena/chainweb-node-client typed js wrapper with fetch to call chainweb-node API endpoints (amongst others https://api.chainweb.com/openapi/pact.html).
+This will probably have some breakdown
+  * api one-to-one mapping of rest endpoints to typed js client
+  * utils functions like that use the api functions to get information
+* @kadena/chainweb-data-client typed js wrapper with fetch to call chainweb-data API endpoints
+* @kadena/pactjs-core low-level generator for pact expressions
+* @kadena/pactjs-cli cli to generate pact contract type definitions and interface to pact client. deployment of contracts, etc
+* @kadena/pactjs-client wrapper around chainweb-node-client with ability to switch environments etc.
+* @kadena/wallet-client client for wallet to sign, connect, retreive account info, etc
+* @kadena/marmalade-client specific client for marmalade/NFTs
```
Comment (alber70g) [inline CONTRIBUTING.md:67]: Did you check these with the updated issue? [Definition of libraries in kadena.js #42](https://github.com/kadena-community/kadena.js/issues/42)

**Review by alber70g — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (Randynamic) [reply .github/CODEOWNERS:11]: Yes when I get access I'll create the teams
short-term sollution

File: CONTRIBUTING.md (hunk 2 above)
Comment (Randynamic) [reply CONTRIBUTING.md:67]: not yet

**Review by ggobugi27 — APPROVED** [review]

**Review by alber70g — APPROVED** [review]

**Pull request merged**